index.js

```diff
@@ -56,20 +58,34 @@ exports.delay = (timeout) => {
 };
 
 exports.focusApplication = async (application, aliases) => {
-  const matching = applicationMatches(application, await exports.getRunningApplications(), aliases);
-  if (matching.length == 0) {
-    return;
+  application = normalizeApplication(application);
+
+  // if we have an exact match without any aliasing, then prioritize that
+  if (applicationMatches(application, await exports.getRunningApplications(), {}).length > 0) {
+    return lib.focusApplication(application);
+  }
+
+  // otherwise, try to focus using the alias map
+  if (aliases && aliases[application]) {
+    application = normalizeApplication(aliases[application]);
   }
 
```

Comment:
Do we need to handle the case where no match is found?

Reply:
I think we're okay in that case, since the C++ call will just find nothing.